ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -305,6 +311,36 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
     const canDeleteNetwork =
       isUnlocked && !isCurrentNetwork && network.chainId !== CHAIN_IDS.MAINNET;
 
+    const onClickCallback = useCallback(() => {
+      handleNetworkChange(network);
+    }, [handleNetworkChange, network]);
+
+    const onDeleteClickCallback = useCallback(() => {
+      dispatch(toggleNetworkMenu());
+      dispatch(
+        showModal({
+          name: 'CONFIRM_DELETE_NETWORK',
+          target: network.chainId,
+          onConfirm: () => undefined,
+        }),
+      );
+    }, [canDeleteNetwork, dispatch, toggleNetworkMenu, showModal, network]);
+
+    const onEditClickCallback = useCallback(() => {
+      dispatch(
+        setEditedNetwork({
+          chainId: network.chainId,
+          nickname: network.name,
+        }),
+      );
+      setActionMode(ACTION_MODES.ADD_EDIT);
+    }, [dispatch, network, setEditedNetwork, setActionMode, ACTION_MODES]);
```

Comment:
`ACTION_MODES` is an enum/constant created once outside of the component? If so then we don't necessarily need it in the dep array.